### PR TITLE
Master 4.1.x/lzma limit/v1

### PR DESCRIPTION
**Pull request by jasonish**
Backport of lzma configuration.

For 4.1.x we disable lzma by default for libhtp.

[PRScript](https://redmine.openinfosecfoundation.org/projects/suricata/wiki/PRscript) output (if applicable):

### CODE REVIEW

**Comment by victorjulien** [comment]
This doesn't pass SV currently as the evader tests that use lzma fail.

File: rules/http-events.rules [hunk 1]
```diff
@@ -52,5 +52,7 @@ alert http any any -> any any (msg:"SURICATA HTTP Request line with leading whit
 alert http any any -> any any (msg:"SURICATA HTTP Request too many encoding layers"; flow:established,to_server; app-layer-event:http.too_many_encoding_layers; flowint:http.anomaly.count,+,1; classtype:protocol-command-decode; sid:2221032; rev:1;)
 alert http any any -> any any (msg:"SURICATA HTTP Request abnormal Content-Encoding header"; flow:established,to_server; app-layer-event:http.abnormal_ce_header; flowint:http.anomaly.count,+,1; classtype:protocol-command-decode; sid:2221033; rev:1;)
 
-# next sid 2221034
 
+alert http any any -> any any (msg:"SURICATA HTTP LZMA reached its memory limit"; flow:established; app-layer-event:http.lzma_memlimit_reached; flowint:http.anomaly.count,+,1; classtype:protocol-command-decode; sid:2221046; rev:1;)
```
Comment (catenacyber) [inline rules/http-events.rules]: The Sid is not in order

Comment (catenacyber) [reply rules/http-events.rules]: Nevermind, Victor tells that the Sid should match between 4.1 and 5.0

**Comment by catenacyber** [comment]
> This doesn't pass SV currently as the evader tests that use lzma fail.

Should we update http-evader-116 to require min version of 5.0 or have a specific suricata.yaml for it ? or something else ?

**Comment by victorjulien** [comment]
> > This doesn't pass SV currently as the evader tests that use lzma fail.
> 
> Should we update http-evader-116 to require min version of 5.0 or have a specific suricata.yaml for it ? or something else ?

I have an updated queued: I'm just passing another 'args' to enable lzma explicitly.

**Comment by victorjulien** [comment]
> > > This doesn't pass SV currently as the evader tests that use lzma fail.
> > 
> > 
> > Should we update http-evader-116 to require min version of 5.0 or have a specific suricata.yaml for it ? or something else ?
> 
> I have an updated queued: I'm just passing another 'args' to enable lzma explicitly.

https://github.com/OISF/suricata-verify/commit/24190564873d39744aec79307971fe275317fb0e

**Pull request merged**